recipe/meta.yaml

```
@@ -56,6 +60,31 @@ outputs:
         convenient.
 
         This package contains utility modules to talk to Avahi with Python easier.
+  - name: avahi
+    script: lib-avahi.sh
+    requirements:
+      build:
+        # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+        # Compilers are named 'c', 'cxx' and 'fortran'.
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - make
+        - pkg-config
+        - intltool
```

I don't think this is actually used?

```suggestion
        - intltool
```